steward/src/cellars.rs

```diff
@@ -157,7 +172,10 @@ pub const ADAPTOR_MORPHO_AAVE_V3_A_TOKEN_COLLATERAL_V1: &str =
 pub const ADAPTOR_MORPHO_AAVE_V3_DEBT_TOKEN_V1: &str = "25a61f771af9a38c10ddd93c2bbab39a88926fa9";
 pub const ADAPTOR_MORPHO_AAVE_V3_P2P_V1: &str = "4fe068caad05b82bf3f86e1f7d1a7b8bbf516111";
 pub const ADAPTOR_UNIV3_V3: &str = "92611574ec9bc13c6137917481dab7bb7b173c9b";
-pub const ADAPTOR_VESTING_SIMPLE_V2: &str = "3b98ba00f981342664969e609fb88280704ac479";
+pub const ADAPTOR_VESTING_SIMPLE_V1_1_DEPLOYMENT1: &str =
+    "3b98ba00f981342664969e609fb88280704ac479";
+pub const ADAPTOR_VESTING_SIMPLE_V1_1_DEPLOYMENT2: &str =
+    "44ce7c941cef04c8e6daea9ac147200e9bb7e7cc";
```

Comment:
This adaptor was deployed to a different address: 0x8a95BBAbb0039480F6DD90fe856c1E0c3D575aA1

Reply:
Replaced it with the correct address.